packages/backend/src/errors/step.ts

```diff
@@ -0,0 +1,12 @@
+import { IStepError } from '@plumber/types'
+
+export default class StepError extends Error {
+  details = {}
```

Comment:
add stronger typing, this is basically typing details as `any`

Reply:
after discussion, we decided to not include details

Reply:
eh why `details` still there